pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -973,8 +993,24 @@ _ReaderContext::Open(const std::string& filePath, std::string* errorLog)
         _SetupInstancing(instances, promotable, &usedRootNames);
     }
 
+    // Re-root so the <defaultPrim> is actually the archive!
+    TfToken abcReRoot;
+    if (args) {
+        const auto reRoot = args->find("abcReRoot");
+        if (reRoot != args->end())
+            abcReRoot = TfToken(reRoot->second);
```

Comment:
You should verify TfIsValidIdentifier() on abcReRoot, and discard and warn if not.